src/apis/onepassword/README.md

```diff
@@ -16,20 +16,46 @@ By default `1password` API type has built in pagination settings and sets the `r
 
 
 ## Example
-```Yaml
+You can customize the endpoints to collect data from by adding extra API configurations under `apis`. 1Password API Docs can be found [here](https://developer.1password.com/docs/connect/connect-api-reference/).
+
+Example configuration:
+
+```yaml
 apis:
-  - name: 1Password example
+  - name: 1Password Audit Events
     type: 1password
     onepassword_bearer_token: <<1PASSWORD_BEARER_TOKEN>>
     url: https://events.1password.com/api/v1/auditevents
     method: POST
-    days_back_fetch: 3
+    days_back_fetch: 7
+    scrape_interval: 5    
+    additional_fields:
+      type: 1password
+      eventType: auditevents
+
+  - name: 1Password Item Usages
+    type: 1password
+    onepassword_bearer_token: <<1PASSWORD_BEARER_TOKEN>>
+    url: https://events.1password.com/api/v1/itemusages
+    method: POST
+    days_back_fetch: 7
+    scrape_interval: 5
+    additional_fields:
+      type: 1password
+      eventType: itemusages
+
+  - name: 1Password Sign In Attempts
+    type: 1password
+    onepassword_bearer_token: <<1PASSWORD_BEARER_TOKEN>>
+    url: https://events.1password.com/api/v1/signinattempts
+    method: POST
+    days_back_fetch: 7
     scrape_interval: 5
     additional_fields:
       type: 1password
-    onepassword_limit: 200
+      eventType: signinattempts
 
 logzio:
-  url: https://listener-eu.logz.io:8071  # for us-east-1 region delete url param (default)
```

Comment:
all the comments in the doc are structured like this, and not with the `<<LISTENER-HOST>>` because the logzio explanation is in the main documentation page.
So I think it's worth to keep it unified